src/exchanges.js

```diff
@@ -31,6 +31,11 @@ let commonRoutingKey = function(options) {
                         'characters aside from - and _ have been stripped.',
       maxSize:          100,
       required:         true,
+    }, {
+      name:             'installationID',
+      summary:          'The installation which had an event.',
```

Comment:
Maybe explain what an intstallation is here a bit more? Even saying something like "Github installation" might help.